### DISCUSSION TITLE
cannot delete zombie etcd pod after accidentally starting rke2-server in my rke2-agent node

### DISCUSSION THREAD

**Discussion by imageschool** [rancher/rke2#3229] [General]
**Environmental Info:**
RKE2 Version: 

**server**
rke2 version v1.22.9+rke2r2 (d7c26a45b92cf3f76c063e93f8c6448fde7b2456)
go version go1.16.14b7

**agent** (Accident happened here)
rke2 version v1.22.6+rke2r1 (2dcedaf55c49ef9b24849c92702bdc12ea589d7a)
go version go1.16.10b7
Node(s) CPU architecture, OS, and Version: 

Cluster Configuration:

**Describe the bug:**

I accidentally mistyped `systemctl restart rke2-server` in my agent node and then to fix it back turned it off by doing`systemctl stop rke2-server`.
Although rke2-server service is stopped, a new etcd pod has been summoned in my agent node and there is literally no way to delete this zombie pod.

This zombie etcd pod is in state 'CrashLoopBackOff', and its priorityClassName is set as system-cluster-critical.

I would really appreciate it if someone can point me in the right direction to delete this evil zombie etcd pod.

I have tried
- `sudo systemctl stop rke2-agent && sudo rm /var/lib/kubelet/cpu_manager_state && sudo kill -9 $(pgrep -f kube) && sudo systemctl restart rke2-agent`
- node drain and following `rke2-killall.sh`
- `etcdctl member list` in master's etcd pod but gave warnings only

**Steps To Reproduce:**

- Installed RKE2: 

**Expected behavior:**

**Actual behavior:**

**Additional context / logs:**

**Comment by brandond** [comment]
I would probably try:
* Stop rke2-agent on the node
* Use `kubectl delete node` to delete the node from the cluster; this will also delete any pods bound to it.
* Start rke2-agent on the node

**Comment by imageschool** [comment]
@brandond thanks for the suggestion.

we wanted to fix it while not deleting the node therefore, we ended up resolving it by manually removing etcd static pod definition in `/var/lib/rancher/rke2/agent/pod-manifests`

**Reply by brandond** [reply]
Ah yes, that would definitely cause it to come back if not cleaned up.
